Psychtoolbox/PsychDemos/DrawFormattedText2Demo.m

```diff
@@ -0,0 +1,138 @@
+% DrawFormattedText2Demo
+%
+% Draws formatted text, shows how to align and center text vertically
+% and/or horizontally, how line-breaks are introduced, how to transform
+% text.
+%
+% Press any key to cycle through different demo displays.
+%
+% see also: PsychDemos, Screen DrawText?, DrawSomeTextDemo, DrawFormattedText2Demo
+
+% 09/15/16    dcn    Wrote it.
+
+try
+    AssertOpenGL;   % check PTB is functioning
```

Comment:
Would use PsychDefaultSetup(1) in new demos, as it includes AssertOpenGL and KbName('UnifyKeynames').
Maybe one should even use PsychDefaultSetup(2) to a normalized 0-1 color range nowadays.

Btw. your way of allowing to change text colors via markup might need to take into account that people don't use the old 0-255 integer color values but the more modern 0.0 - 1.0 floating point color range.